README.md

```diff
@@ -145,9 +144,16 @@ npm install --save jupyterlab
 git clone https://github.com/jupyter/jupyterlab.git
 cd jupyterlab
 npm install
-npm run build:all
+npm run build
+cd jupyterlab  # into the server extension
+npm install
+npm run build
 ```
 
+Note: If using an earlier version of node (<4), it is advised that you
+run `npm dedupe` after `npm install`, since older versions of node
+did not install dependencies maximally flat, which can cause version conflicts.
```

Comment:
To be picky "did not" is past tense, but the rest of the sentence is in present tense, so it probably should be "do not"....

Feel free to ignore :).